docs/tests/jest.md

```diff
@@ -21,44 +21,58 @@
 
 Additional methods and information to debug in VS Code can be found [here](https://jestjs.io/docs/troubleshooting#debugging-in-vs-code)
 
-### Debugging Jest on Chrome DevTools
+## Running a Jest test on a node server and inspecting in Chrome DevTools
 
 1. Run Jest using Node with the V8 Inspector
 
     ```bash
-    > node --inspect ./node_modules/.bin/jest --watch -i <path>
+    > yarn jest:node <path>
```

Comment:
Should be this ？
```suggestion
    > yarn test:unit:jest:watch" <path>
```

Reply:
thank you @DDDDDanica! yes. updated without the end quote [1f2c568](https://github.com/MetaMask/metamask-extension/pull/20683/commits/1f2c56810e2b5b0a82cd7401ec4761fb56415967)